demos/bookstore/app/router.ts

```diff
@@ -28,10 +28,47 @@ middleware.push(formData({ uploadHandler }))
 middleware.push(methodOverride())
 middleware.push(storeContext())
 
+middleware.push(
+  staticFiles('./public/root', {
```

Comment:
This is where `staticFiles` really shines, IMO. In the "global" middleware case.

Trying to re-use `staticFiles` as an inline middleware on a specific route feels a little awkward because we have to pass the param from the route to the middleware somehow. I think I'd prefer to have a more direct approach in a route handler, something like:

```tsx
router.get('/assets/*path', (context) => {
  let file = findFile('./assets', context.params.path)
  if (file != null) return sendFile(file, context) // sendFile could handle ETag, Ranges, etc.
  return new Response('Not Found', { status: 404 })
})
```